spec/recurly/subscription_spec.rb

```diff
@@ -351,4 +351,38 @@
       redemptions.all? { |r| r.is_a? Redemption }.must_equal true
     end
   end
+
+  describe '#update_attributes' do
+    describe 'when the plan code does change' do
+      it 'sends all updated atributes to the server' do
+        stub_api_request :get, 'subscriptions/abcdef1234567890', 'subscriptions/show-200'
+
+        subscription = Subscription.find 'abcdef1234567890'
+
+        stub_request(:put, "https://api_key:@api.recurly.com/v2/subscriptions/abcdef1234567890").
```

Comment:
Yeah I did the same thing when I tested it. I modified the `stub_api_request` method to handle it but it was ugly. I think this is good for now. I'm going to do some refactoring of the tests soon.